email-filters.html

```diff
@@ -0,0 +1,166 @@
+                        <a class="nav-link" href="modules.html">Mods</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="license.html">License</a>
+                    </li>
+                    <li class="nav-item">
+                        <!--<a class="nav-link" href="tests.html">Test</a>-->
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="documentation.html">Documentation</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="https://unencumberedbyfacts.com" target="_blank">Blog</a>
+                    </li>
+                    <li class="nav-item">
+                        <a onclick="document.getElementsByClassName('navbar-collapse')[0].style.display='none';"
+                           class="nav-link" href="contribute.html">Contribute</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+</header>
+
+<section class="content-section container">
+    <h2>Email filters</h2>
+    <p>Cypht now supports Sieve, a powerful email filtering language that gives you complete control over your inbox. With Sieve, you can create complex filters to sort, move, delete, and flag your emails automatically. You can also use Sieve to create custom notifications and alerts.</p>
+    <p>Here are just a few of the things you can do with Sieve:</p>
+    <hr>
+    <ul>
+        <li>
+            Filter out spam and unwanted emails. Sieve filters can be used to filter out spam and unwanted emails based on their sender, subject, or content.
+        </li>
+        <hr>
+        <li>
+            Organize your inbox automatically. Sieve filters can be used to automatically move or copy your emails into different folders based on their sender, subject, or content.
+        </li>
+        <hr>
+        <li>
+            Forward important emails to specific people. Sieve filters can be used to automatically forward important emails to specific people, such as your manager or your assistant.
+        </li>
+        <hr>
+        <li>
+            Sieve filters can be used to create custom notifications and alerts for important emails, such as those from your boss or clients. Unlike simply forwarding the email, these notifications and alerts can provide immediate and specific information, ensuring you stay updated without needing to sift through your inbox.
+        </li>
+        <hr>
+    </ul>
+    <br/>
+    <p> To get started with Sieve in Cypht, simply go to your config file (hm3.ini for Cypht 1.4.x or .env for Cypht 2.0.x) and enable the Sieve filter engine by enabling modules[ ]=sievefilters or by adding sievefilters on CYPHT_MODULES variable like:
+
+        <pre>modules[]=sievefilters #For Cypht 1.4.x
+CYPHT_MODULES="sievefilters" #For Cypht 2.0.x</pre>
```

Comment:
Is it necessary to run a script after? (I hope not)

I don't understand why Sieve is not just a checkbox in the admin panel. It's a pretty standard feature.

Reply:
We'll may be suggest it here: [Cypht issues](https://github.com/cypht-org/cypht/issues), but as a module in Cypht that is how we enable it as all other modules in Cypht.